frontend/document/credocument.lua

```diff
@@ -1512,6 +1517,7 @@ function CreDocument:setupCallCache()
             elseif name:sub(1,6) == "enable" then add_reset = true
             elseif name == "zoomFont" then add_reset = true -- not used by koreader
             elseif name == "resetCallCache" then add_reset = true
+            elseif name == "resetBufferCache" then add_buffer_trash = true
```

Comment:
Right, just move it below with the other add_buffer_trash after the findText one.